src/Common/src/Interop/Windows/sspicli/SSPIWrapper.cs

```diff
@@ -484,34 +360,24 @@ private unsafe static int EncryptDecryptHelper(OP op, SSPIInterface secModule, S
                         // Backup validate the new sizes.
                         if (iBuffer.offset < 0 || iBuffer.offset > (iBuffer.token == null ? 0 : iBuffer.token.Length))
                         {
-                            if (GlobalLog.IsEnabled)
-                            {
-                                GlobalLog.AssertFormat("SSPIWrapper::EncryptDecryptHelper|'offset' out of range.  [{0}]", iBuffer.offset);
-                            }
-
-                            Debug.Fail("SSPIWrapper::EncryptDecryptHelper|'offset' out of range.  [" + iBuffer.offset + "]");
+                            NetEventSource.Fail(null, $"'offset' out of range.  [{iBuffer.offset}]");
                         }
 
                         if (iBuffer.size < 0 || iBuffer.size > (iBuffer.token == null ? 0 : iBuffer.token.Length - iBuffer.offset))
                         {
-                            if (GlobalLog.IsEnabled)
-                            {
-                                GlobalLog.AssertFormat("SSPIWrapper::EncryptDecryptHelper|'size' out of range.  [{0}]", iBuffer.size);
-                            }
-
-                            Debug.Fail("SSPIWrapper::EncryptDecryptHelper|'size' out of range.  [" + iBuffer.size + "]");
+                            NetEventSource.Fail(null, $"'size' out of range.  [{iBuffer.size}]");
                         }
                     }
 
-                    if (errorCode != 0 && NetEventSource.Log.IsEnabled())
+                    if (errorCode != 0 && NetEventSource.IsEnabled)
```

Comment:
I suppose it doesn't really matter.  

I guess I'd like to adhere to a principle that says: Any code that's only needed for tracing should be inside an IsEnabled check, implicitly or explicitly.  And here, the errorCode != 0 test is only needed for tracing.  I realize that's being rather nitty.
I like the idea of consistently keeping any code that's not needed in